pkg/disk/partition_table.go

```diff
@@ -715,7 +858,7 @@ func (pt *PartitionTable) ensureLVM() error {
 
 		// create root logical volume on the new volume group with the same
 		// size and filesystem as the previous root partition
-		_, err := vg.CreateLogicalVolume("root", part.Size, filesystem)
+		_, err := vg.CreateLogicalVolume("rootlv", part.Size, filesystem)
```

Comment:
This commit doesn't affect the default image definitions, but it does change the behaviour of `vg.CreateLogicalVolume()`.  It also doesn't affect the behaviour of `CreateMountpoint()`.

I had to make this change so that I can use `CreateLogicalVolume()` without having the lv name change internally in the function.  The function is exported, but it's not used in osbuild-composer directly, so it shouldn't have any effect on the behaviour there.